### PR TITLE
i18n(zh-cn): translate /guides/cms/caisy.mdx

### PR DESCRIPTION
**Pull request by Lilian97**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- add i18n(zh-cn): translate `/guides/cms/caisy.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/caisy.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Caisy & Astro
+description: 使用 Caisy 作为 CMS 向你的 Astro 项目添加内容
+type: cms
+service: Caisy
+i18nReady: true
+stub: true
+---
+
+[Caisy](https://caisy.io/) 是一个无头 CMS ，它公开了 GraphQL API 来存取内容。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/caisy.mdx:10]:
```suggestion
[Caisy](https://caisy.io/) 是一个无头（headless） CMS ，它公开了 GraphQL API 来存取内容。
```

File: src/content/docs/zh-cn/guides/cms/caisy.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## 将 Caisy CMS 与 Astro 结合使用
+
+使用 `graphql request` 和 Caisy 的 Astro 富文本渲染器来获取你的 CMS 数据，并在 Astro 页面上显示你的内容：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/caisy.mdx:14]:
```suggestion
使用 `graphql-request` 和 Caisy 的 Astro 富文本渲染器来获取你的 CMS 数据，并在 Astro 页面上显示你的内容：
```

**Review by liruifengv — APPROVED** [review]
A little change. LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good work peeps 💜

**Pull request merged**